ee/modules/160-multitenancy-manager/templates/user-resources/default-project-template.yaml

```diff
@@ -128,7 +128,7 @@ spec:
     apiVersion: deckhouse.io/v1alpha1
     kind: AuthorizationRule
     metadata:
-      name: {{ $administrator.name }}
+      name: {{ $administrator.name | replace "@" "-" }}
```

Comment:
The idea was to cover more cases and slugify other special characters like spaces.
In previous implementation, we used the slugify helper:
https://github.com/deckhouse/deckhouse/blob/cff98f6efc422621e5e9d26b4ff9930ccf5dc47f/ee/modules/160-multitenancy-manager/templates/user-resources/templates/user-resources-templates.yaml#L106

Reply:
does it mean it's a an eligible case to a group like below
```
root@dev-master-0:~# kubectl  get groups.deckhouse.io  
NAME             GROUP
administrators   administrator@s
```
and match it by "administrator@s", not "administrators"?

Reply:
we (I) thought the problem was solely about having to provide a string in a email notation  if we assign the project to a user, which would result in having ineligible, from k8s point of view, resource name with '@' character.

Reply:
The problem is with the metadata.name field, for groups any string is valid